sentry-rails/spec/sentry/rails/breadcrumbs/active_support_breadcrumbs_spec.rb

```diff
@@ -0,0 +1,91 @@
+require "spec_helper"
```

Comment:
is this a copy of the old test file? if it is, shouldn't the old one get deleted?

Reply:
Oups 🤦🏻  
Yes it's a copy, thanks it's done!